recipes/ncurses/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "5.9" %}
+
+package:
+  name: ncurses
+  version: {{ version }}
+
+source:
+  fn: ncurses-{{ version }}.tar.gz
+  url: http://ftp.gnu.org/pub/gnu/ncurses/ncurses-{{ version }}.tar.gz
+  md5: 8cb9c412e5f2d96bc6f459aa8c6282a1
+  patches:
+    # http://lists.gnu.org/archive/html/bug-ncurses/2011-04/txtkWQqiQvcZe.txt
+    - fix.patch
+
+build:
+  skip: true  # [win]
+  number: 0
+  detect_binary_files_with_prefix: true
+
+test:
+  files:
+    - run_test.sh         # [osx]
+    - run_test.py         # [osx]
+  commands:
+    - exit $(test -f ${PREFIX}/lib/libncurses.a)
+    - exit $(test -f ${PREFIX}/lib/libtinfo.a)
+    - exit $(test -f ${PREFIX}/lib/libform.a)
+    - exit $(test -f ${PREFIX}/lib/libmenu.a)
+    - exit $(test -f ${PREFIX}/lib/libpanel.a)
+
+    - bash run_test.sh    # [osx]
+    - python run_test.py  # [osx]
```

Comment:
I had the impression that files named `run_test.{sh,py}` would be automatically discovered and run by `conda-build`. Is that necessary because you have both instead of one? If so I consider this a conda-build bug.